src/Docs/Resources/current/2-internals/4-plugins/010-plugin-quick-start.md

```diff
@@ -123,7 +123,7 @@ Read [here](./050-plugin-information.md) for more information about the content
 Now, that you've created the two necessary plugin files, you're able to install the plugin.
 This is done using one of the [plugin commands](./030-plugin-commands.md).
 
-Starting in your **project root** directory, run the command `bin/console plugin:install --activate --clearCache PluginQuickStart` to install and activate the plugin.
+Starting in your **project root** directory, check if your plugin is detected run `bin/console plugin:list`. When your new plugin is missing, execute the command `bin/console plugin:refresh`. Then run the command `bin/console plugin:install --activate --clearCache PluginQuickStart` to install and activate the plugin.
```

Comment:
Would you agree with this version as well? :-)

```suggestion
Starting in your **project root** directory, check if your plugin is already known to Shopware by running the command `bin/console plugin:list`. If your new plugin is missing, execute the command `bin/console plugin:refresh`. Afterwards run the command `bin/console plugin:install --activate --clearCache PluginQuickStart` to install and activate the plugin.
```

Reply:
Shure.